crates/op-rbuilder/src/payload_builder_vanilla.rs

```diff
@@ -124,13 +124,18 @@ where
         + Unpin
         + 'static,
     <Pool as TransactionPool>::Transaction: OpPooledTx,
+    Evm: ConfigureEvm<
+            Primitives = PrimitivesTy<Node::Types>,
+            NextBlockEnvCtx = OpNextBlockEnvAttributes,
+        > + 'static,
 {
     type PayloadBuilder = OpPayloadBuilderVanilla<Pool, Node::Provider>;
 
     async fn build_payload_builder(
         self,
         ctx: &BuilderContext<Node>,
         pool: Pool,
+        _evm_config: Evm,
```

Comment:
why add evm config if it is not used?

Reply:
Because they changed the method signature in the new reth, and we implement reth trait

Reply:
what did they add it for? should our builder use the evm config from the arg as well?

Reply:
They needed it to access evm state for some bug with extradata for Ethereum block building